### DISCUSSION TITLE
"Look up profile" icon

### DISCUSSION THREAD

**Discussion by apombo-ux** [twilio-labs/paste#3164] [Request: Icon] [answered]
### Description

Hi there,
My team is adding a new widget to Studio that allows users to look up customer profiles from Segment or another connected CDP. I don't see an icon in the Paste library that represents 'profile,' but this one from Streamline fits well: https://www.streamlinehq.com/icons/streamline-regular?search=target+human+resources

<img width="827" alt="image" src="https://user-images.githubusercontent.com/98772241/232101633-24d21e90-d018-44c2-a944-80bc134c24ce.png">

It'd be a UI Icon. Here are examples of where it would be used in Studio:
![image](https://user-images.githubusercontent.com/98772241/232100032-9605dddc-da0b-4205-aed8-f06fd429c028.png)

![image](https://user-images.githubusercontent.com/98772241/232099845-68a9e16a-f3a3-4e33-9b6c-0defd339a6c9.png)

### Link to icon

https://www.streamlinehq.com/icons/streamline-regular?search=target+human+resources

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @apombo-ux ! Are there other use cases outside of this Studio flow where this icon would be used, especially by other teams? I'm going to suggest a few options here before we consider adding this new icon:

1. Is it possible to use the Segment icon here instead? If a user can connect another CDP besides Segment, I'm guessing not, but I figured I'd suggest it just in case.
2. Is it possible to use the Search icon instead?

If these options don't work for you, your team could consider using the SVG-to-React library to create and maintain the icon component within your team's own library: https://paste.twilio.design/core/libraries#libraries Unfortunately we aren't able to export from Streamline right now, so you'll also have to recreate the SVG yourself, too.

**Reply by apombo-ux** [reply]
Hi Sarah-
There aren't any other places this icon would be used that I'm aware of. 

1. Right, the requirement is for the widget to not be Segment-specific, so the Segment brand icon wouldn't work here.
2. We'd been focusing on options that make the 'profile' part of the widget purpose visually apparent, so I hadn't considered just the search icon. I'll give it a try with just that and see how that feels.

If we do end up going with this icon though, thanks for the info about how to use it in our own library!

**Reply by serifluous** [reply]
Sounds good, let us know what you decide to go with!

**Reply by apombo-ux** [reply]
We're going to go ahead use the custom icon and will maintain it ourselves. Thanks again for the guidance!
